worlds/sc2/ItemGroups.py

```diff
@@ -77,24 +76,486 @@
+    hots_units
+    + zerg_buildings
+    + kerrigan_abilities
+    + hots_mutations
+    + hots_strains
+    + hots_global_upgrades
+    + zerg_generic_upgrades
+)
+
+
+# Protoss
+item_name_groups[ItemGroupNames.PROTOSS_ITEMS] = protoss_items = [
+    item_name for item_name, item_data in Items.item_table.items()
+    if item_data.race == SC2Race.PROTOSS
+]
+item_name_groups[ItemGroupNames.PROTOSS_UNITS] = protoss_units = [
+    item_name for item_name, item_data in Items.item_table.items()
+    if item_data.type in (Items.ProtossItemType.Unit, Items.ProtossItemType.Unit_2)
+]
+item_name_groups[ItemGroupNames.PROTOSS_GENERIC_UPGRADES] = protoss_generic_upgrades = [
+    item_name for item_name, item_data in Items.item_table.items()
+    if item_data.type == Items.ProtossItemType.Upgrade
+]
+item_name_groups[ItemGroupNames.LOTV_UNITS] = lotv_units = [
+    ItemNames.ZEALOT, ItemNames.CENTURION, ItemNames.SENTINEL,
+    ItemNames.STALKER, ItemNames.DRAGOON, ItemNames.ADEPT,
+    ItemNames.SENTRY, ItemNames.HAVOC, ItemNames.ENERGIZER,
+    ItemNames.HIGH_TEMPLAR, ItemNames.DARK_ARCHON, ItemNames.ASCENDANT,
+    ItemNames.DARK_TEMPLAR, ItemNames.AVENGER, ItemNames.BLOOD_HUNTER,
+    ItemNames.IMMORTAL, ItemNames.ANNIHILATOR, ItemNames.VANGUARD,
+    ItemNames.COLOSSUS, ItemNames.WRATHWALKER, ItemNames.REAVER,
+    ItemNames.PHOENIX, ItemNames.MIRAGE, ItemNames.CORSAIR,
+    ItemNames.VOID_RAY, ItemNames.DESTROYER, ItemNames.ARBITER,
+    ItemNames.CARRIER, ItemNames.TEMPEST, ItemNames.MOTHERSHIP,
+]
+item_name_groups[ItemGroupNames.PROPHECY_UNITS] = prophecy_units = [
+    ItemNames.ZEALOT, ItemNames.STALKER, ItemNames.HIGH_TEMPLAR, ItemNames.DARK_TEMPLAR,
+    ItemNames.OBSERVER, ItemNames.IMMORTAL, ItemNames.COLOSSUS,
+    ItemNames.PHOENIX, ItemNames.VOID_RAY, ItemNames.CARRIER,
+]
+item_name_groups[ItemGroupNames.PROPHECY_BUILDINGS] = prophecy_buildings = [
+    ItemNames.PHOTON_CANNON,
+]
+item_name_groups[ItemGroupNames.GATEWAY_UNITS] = gateway_units = [
+    ItemNames.ZEALOT, ItemNames.CENTURION, ItemNames.SENTINEL, ItemNames.SUPPLICANT,
+    ItemNames.STALKER, ItemNames.INSTIGATOR, ItemNames.SLAYER,
+    ItemNames.SENTRY, ItemNames.HAVOC, ItemNames.ENERGIZER,
+    ItemNames.DRAGOON, ItemNames.ADEPT, ItemNames.DARK_ARCHON,
+    ItemNames.HIGH_TEMPLAR, ItemNames.SIGNIFIER, ItemNames.ASCENDANT,
+    ItemNames.DARK_TEMPLAR, ItemNames.AVENGER, ItemNames.BLOOD_HUNTER,
+]
+item_name_groups[ItemGroupNames.ROBO_UNITS] = robo_units = [
+    ItemNames.WARP_PRISM, ItemNames.OBSERVER,
+    ItemNames.IMMORTAL, ItemNames.ANNIHILATOR, ItemNames.VANGUARD,
+    ItemNames.COLOSSUS, ItemNames.WRATHWALKER,
+    ItemNames.REAVER, ItemNames.DISRUPTOR,
+]
+item_name_groups[ItemGroupNames.STARGATE_UNITS] = stargate_units = [
+    ItemNames.PHOENIX, ItemNames.MIRAGE, ItemNames.CORSAIR,
+    ItemNames.VOID_RAY, ItemNames.DESTROYER,
+    ItemNames.SCOUT, ItemNames.TEMPEST,
+    ItemNames.CARRIER, ItemNames.MOTHERSHIP,
```

Comment:
In order to respect command card slots, the Mothership is in Scout/Tempest slot, not Carrier's